1.ts

```diff
@@ -0,0 +1,12 @@
+@Component({
```

Comment:
Style guide: properties in @Component decorator should be sorted by next order: false